src/components/table.ts

```diff
@@ -9,7 +9,7 @@ export module Components {
     wantsHeightArr: boolean[];
   }
 
-  export type _IterateLayoutResult = {
+  type _IterateLayoutResult = {
```

Comment:
Should we just inline this and remove the type? It's only being used in one place